### PR TITLE
feat: implemented unit conversion

### PR DESCRIPTION
**Pull request by muthukumarimoorthi**
implemented a dedicated function for the conversion of consumption units cost.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -103,23 +103,23 @@ <h2 class="text-[#97F85C] font-bold text-3xl">
           <div class="flex flex-col gap-y-2">
             <div class="flex items-center gap-x-3 bg-gray-500/20 rounded-lg">
               <span class="bg-[#1C4349] text-[#97F85C] p-2 font-semibold rounded-lg border-r-4 border-[#97F85C]">Account Number</span>
-              <span>1122334455</span>
+              <span id = "account-number">1122334455</span>
```
Comment (mshajid) [inline index.html:106]: @muthukumarimoorthi : **Rework** on the ID's you have given.

**Review by mshajid — COMMENTED** [review]
@muthukumarimoorthi : Kindly work on the reviews given for as your reference.

File: index.html (hunk 1 above)
Comment (muthukumarimoorthi) [reply index.html]: @mshajid: i have changed id names and unwanted logs

Comment (Bearcin46) [reply index.html]: @muthukumarimoorthi Well done! . @mshajid She had completed the work perfectly.

Comment (mshajid) [reply index.html]: @muthukumarimoorthi @Bearcin46 : Thanks 👯

**Pull request merged**